src/components/Calculator.js

```diff
@@ -0,0 +1,78 @@
+import './stylesheet/calculator.css';
+
+function ResultLine() {
+  return (
+    <div className="line result-line">
+      <p className="result-line-text">0</p>
+    </div>
+  );
+}
+
+function FirstLine() {
+  return (
+    <div className="line first-line">
+      <button className="first-line-button" type="button">AC</button>
+      <button className="first-line-button" type="button">+-</button>
+      <button className="first-line-button" type="button">%</button>
+      <button className="first-line-button last" type="button">+</button>
+    </div>
+  );
+}
+
+function SecondLine() {
+  return (
+    <div className="line second-line">
+      <button className="second-line-button" type="button">7</button>
+      <button className="second-line-button" type="button">8</button>
+      <button className="second-line-button" type="button">9</button>
+      <button className="second-line-button last" type="button">x</button>
+    </div>
+  );
+}
+
+function ThirdLine() {
+  return (
+    <div className="line third-line">
+      <button className="third-line-button" type="button">4</button>
+      <button className="third-line-button" type="button">5</button>
+      <button className="third-line-button" type="button">6</button>
+      <button className="third-line-button last" type="button">-</button>
+    </div>
+  );
+}
+
+function FourthLine() {
+  return (
+    <div className="line fourth-line">
+      <button className="fourth-line-button" type="button">1</button>
+      <button className="fourth-line-button" type="button">2</button>
+      <button className="fourth-line-button" type="button">3</button>
+      <button className="fourth-line-button last" type="button">+</button>
+    </div>
+  );
+}
+
+function FifthLine() {
+  return (
+    <div className="line fifth-line">
+      <button className="fifth-line-button first" type="button">0</button>
+      <button className="fifth-line-button" type="button">.</button>
+      <button className="fifth-line-button last" type="button">=</button>
+    </div>
+  );
+}
+
```

Comment:
- [ ] @DevaWinner Great work creating the interface for your calculator, however, its not good practice to have more than one component in a single file, I would recommend you create files for these components and import them into the Clculatir component, this will make your code clear, easier to read and easier to scale in future.

Reply:
- [ ] [optional] @DevaWinner Kindly consider increasing the size of the calculator, especially the width.

Reply:
Hello @danmainah,

Thank you for reviewing my code and suggesting these changes.

I have made the necessary changes and corrections

## Changes Implemented

- [x] I have removed the boilerplate items from the public folder to ensure a clean project structure.
- [x] I have refactored the component structure by moving the button component to a separate file and importing them to follow best practices and improve code readability.
- [x] I have increased the size of the calculator, the width from 20% to 30% and the height from 50% to 60% for better visibility and user experience.

Once again, thank you for this suggested changes.

Cheers 💯